src/graphics/graphics-device.js

```diff
@@ -650,16 +650,20 @@ class GraphicsDevice extends EventHandler {
         } else {
             this.textureFloatRenderable = false;
         }
-        if (this.extTextureHalfFloat) {
+
```

Comment:
```suggestion
        // two extensions allow us to render to half float buffers
        if (this.extColorBufferHalfFloat) {
            this.textureHalfFloatRenderable = !!this.extColorBufferHalfFloat;
        } else if (this.extTextureHalfFloat) {
            if (this.webgl2) {
                // EXT_color_buffer_float should affect both float and halffloat formats
                this.textureHalfFloatRenderable = !!this.extColorBufferFloat;
            } else {
                // Manual render check for half float
                this.textureHalfFloatRenderable = testRenderable(gl, this.extTextureHalfFloat.HALF_FLOAT_OES);
            }
        } else {
            this.textureFloatRenderable = false;
        }
```

Reply:
NB. Not sure if the suggestion above will correctly merge into the PR

Reply:
And its just a suggestion - as I find it a bit more readable.

Reply:
I think you meant to multi-select the lines of code this suggestion applies to.

Reply:
yep, good point. I had those first two conditions the other way before, and this could not work .. but now that I swapped them, good suggestion.